docs/pages/includes/metrics.mdx

```diff
@@ -1,59 +1,67 @@
+| `audit_server_open_files`                            | gauge       | Teleport Audit Log  | Number of open audit files.                                                                        |
+| `auth_generate_requests_throttled_total`             | counter     | Teleport Auth       | Number of throttled requests to generate new server keys.                                          |
+| `auth_generate_requests_total`                       | counter     | Teleport Auth       | Number of requests to generate new server keys.                                                    |
+| `auth_generate_requests`                             | gauge       | Teleport Auth       | Number of current generate requests.                                                               |
+| `auth_generate_seconds`                              | histogram   | Teleport Auth       | Latency for generate requests.                                                                     |
+| `backend_batch_read_requests_total`                  | counter     | cache               | Number of read requests to the backend.                                                            |
+| `backend_batch_read_seconds`                         | histogram   | cache               | Latency for batch read operations.                                                                 |
+| `backend_batch_write_requests_total`                 | counter     | cache               | Number of batch write requests to the backend.                                                     |
+| `backend_batch_write_seconds`                        | histogram   | cache               | Latency for backend batch write operations.                                                        |
+| `backend_read_requests_total`                        | counter     | cache               | Number of read requests to the backend.                                                            |
+| `backend_read_seconds`                               | histogram   | cache               | Latency for read operations.                                                                       |
+| `backend_requests`                                   | counter     | cache               | Number of write requests to the backend.                                                           |
+| `backend_write_seconds`                              | histogram   | cache               | Latency for backend write operations.                                                              |
+| `cluster_name_not_found_total`                       | counter     | Teleport Auth       | Number of times a cluster was not found.                                                           |
+| `dynamo_requests_total`                              | counter     | DynamoDB            | Total number of requests to the DynamoDB API.                                                            |
```

Comment:
The `|` on lines 21 and 22 should be moved further left.

Reply:
I could just run this whole thing through a markdown table formatter if the author is OK with it.

Reply:
fixed in https://github.com/gravitational/teleport/pull/26331/commits/daf76446215db2fc7ce2f7088ec65a3aa7f41aad